src/MOI_wrapper/scaled_psd_cone_bridge.jl

```diff
@@ -19,27 +27,27 @@ function MOI.dimension(x::ScaledPSDCone)
     return div(x.side_dimension * (x.side_dimension + 1), 2)
 end
 
-struct ScaledPSDConeBridge{T,G} <: MOI.Bridges.Constraint.SetMapBridge{
+struct ScaledPSDConeBridge{T,F} <: MOI.Bridges.Constraint.SetMapBridge{
     T,
     ScaledPSDCone,
-    MOI.PositiveSemidefiniteConeTriangle,
-    MOI.VectorAffineFunction{T},
-    G,
+    MOI.ScaledPositiveSemidefiniteConeTriangle,
+    F,
+    F,
 }
-    constraint::MOI.ConstraintIndex{MOI.VectorAffineFunction{T},ScaledPSDCone}
+    constraint::MOI.ConstraintIndex{F,ScaledPSDCone}
```

Comment:
If we're going to support arbitrary functions, then we need to add tests for at least `MOI.VectorOfVariables`.

Reply:
Fair point